internal/config/path.go

```diff
@@ -16,6 +17,8 @@ const (
 	xdgStateHome   = "XDG_STATE_HOME"
 )
 
+var errTiledUsage = errors.New("tilde in path is not supported, use $HOME instead")
```

Comment:
```suggestion
var errTildeUsage = errors.New("tilde in path is not supported, use $HOME instead")
```

Minor typo suggestion